commands/serverless.go

```diff
@@ -361,6 +373,24 @@ func RunServerlessStatus(c *CmdConfig) error {
 	if err != nil || checkNS != creds.Namespace {
 		return do.ErrServerlessNotConnected
 	}
+	// Before doing the normal display, check for the --credentials flag, which requests return of the
+	// 'creds' value as JSON.
+	credentials, _ := c.Doit.GetBool(c.NS, "credentials")
+	if credentials {
+		type showCreds struct {
+			Auth      string `json:auth`
```

Comment:
@joshuaauerbachwatson can we also include the path to the credentials here ? It would be very useful when running the deployer tests.

Reply:
@ddebarros I have added the path to the credentials output.  To do this cleanly I added to the `ServerlessService` which in turn required regenerating the mocks.  This led me to thinking whether we should expand unit tests to cover these paths, so I may do that also before requesting review again.